src/deps/uws/socket.zig

```diff
@@ -317,18 +317,18 @@ pub fn NewSocketHandler(comptime is_ssl: bool) type {
             }
         }
 
-        pub fn write(this: ThisSocket, data: []const u8, msg_more: bool) i32 {
+        pub fn write(this: ThisSocket, data: []const u8, msg_more: bool, error_: ?*i32) i32 {
```

Comment:
This is un-ziglike to use. Maybe let's switch it to use a JSC.Maybe like our other system calls and to make the migration easier, add a new `write` method that returns the error?